[2022-11-04T19:02:22.055+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-17T00:00:00+00:00 [queued]>
[2022-11-04T19:02:22.080+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-17T00:00:00+00:00 [queued]>
[2022-11-04T19:02:22.082+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:02:22.084+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-04T19:02:22.085+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:02:22.133+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): TASK1> on 2022-05-17 00:00:00+00:00
[2022-11-04T19:02:22.146+0000] {standard_task_runner.py:55} INFO - Started process 4742 to run task
[2022-11-04T19:02:22.151+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'orquestacion', 'TASK1', 'scheduled__2022-05-17T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/5.1-orquestacion.py', '--cfg-path', '/tmp/tmpx0cczuy9']
[2022-11-04T19:02:22.152+0000] {standard_task_runner.py:83} INFO - Job 45: Subtask TASK1
[2022-11-04T19:02:22.240+0000] {task_command.py:376} INFO - Running <TaskInstance: orquestacion.TASK1 scheduled__2022-05-17T00:00:00+00:00 [running]> on host c7d12af55512
[2022-11-04T19:02:22.353+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=orquestacion
AIRFLOW_CTX_TASK_ID=TASK1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-17T00:00:00+00:00
[2022-11-04T19:02:22.355+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-04T19:02:22.358+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "SLEEP 2 && ECHO 'TASK1'"]
[2022-11-04T19:02:22.375+0000] {subprocess.py:86} INFO - Output:
[2022-11-04T19:02:22.378+0000] {subprocess.py:93} INFO - /bin/bash: line 1: SLEEP: command not found
[2022-11-04T19:02:22.379+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-11-04T19:02:22.395+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-11-04T19:02:22.402+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=orquestacion, task_id=TASK1, execution_date=20220517T000000, start_date=20221104T190222, end_date=20221104T190222
[2022-11-04T19:02:22.423+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 45 for task TASK1 (Bash command failed. The command returned a non-zero exit code 127.; 4742)
[2022-11-04T19:02:22.444+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-11-04T19:02:22.492+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
